thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/lib/FunctionReplayRunnable.java

```diff
@@ -37,6 +37,7 @@
   private Long functionAutotuneConfigId;
   private boolean speedUp;
   private boolean selfKill;
+  private long clonedFunctionId;
```

Comment:
The only usage here is to do replay once using the input configuration. Hence only one clone function Id will be generated. Modify its name to lastClonedFunctionId.